schema/artist/index.js

```diff
@@ -111,6 +111,23 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      formatted_nationality_and_birthday: {
+        type: GraphQLString,
+        resolve: ({ birthday, nationality }) => {
+          let formatted_bday = (!isNaN(birthday) && birthday) ? 'b. ' + birthday : birthday;
+
+          if (formatted_bday && formatted_bday.toLowerCase().includes('born')) {
```

Comment:
You don’t really need that check if the string includes `born`, because the `formatted_bday.replace(…)` call should only replace it if it exists anyway.